### PR TITLE
#28 Implement SecurityConfig

### LINKED ISSUE #28: security config

### PR DESCRIPTION
**Pull request by ilisau**
In this PR I implemented SecurityConfig with security methods annotations and close #28

### CODE REVIEW

File: src/main/java/com/taskviewer/api/web/security/jwt/SecurityRulesHandler.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+@Component("securityRulesHandler")
+@RequiredArgsConstructor
+public class SecurityRulesHandler {
```
Comment (h1alexbel) [inline src/main/java/com/taskviewer/api/web/security/jwt/SecurityRulesHandler.java:17]: good idea, but make it via the interface

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Review by h1alexbel — APPROVED** [review]

**Review by h1alexbel — COMMENTED** [review]
@ilisau fix imports

**Comment by ilisau** [comment]
@rultor merge

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@ilisau thanks!

**Pull request merged**